platform-sdk/swirlds-state-api/src/testFixtures/java/com/swirlds/state/test/fixtures/StringRecord.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
```

Comment:
Only the current year applies to new files
```suggestion
 * Copyright (C) 2024 Hedera Hashgraph, LLC
```

Reply:
Done